packet_parse/bt_over_usb_wireshark_dissector.lua

```diff
@@ -0,0 +1,175 @@
+  pkt.cols.protocol = p_jc_btusb.name
+ 
+  subtree = root:add(p_jc_btusb, buf(0))
+  --Joy-Con packet
+  if buf(1,1):uint() == 0x20
+    then 
+      subtree:add(f_src, buf(0,1))
+      subtree:add(f_inputid, buf(9,1))
+      subtree:add(f_timming, buf(10,1))
+
+      local switch_subcmdack = buf(22,2):uint()
+
+      if switch_subcmdack == 0x8202 then
+        pkt.cols.info = "Device info"
+        if buf(26,1):uint() == 0x01 then
+          subtree:add(f_subcommandjoy, buf(23,1), buf(23,1):uint(), string.format("Subcmd reply: 0x02 [Device info], Left Joy-Con, FW Version: %X.%02x, MAC Addr: %02x:%02x:%02x:%02x:%02x:%02x", buf(24,1):uint(), buf(25,1):uint(), buf(28,1):uint(), buf(29,1):uint(), buf(30,1):uint(), buf(31,1):uint(), buf(32,1):uint(), buf(33,1):uint()))
+        elseif buf(26,1):uint() == 0x02 then
+          subtree:add(f_subcommandjoy, buf(23,1), buf(23,1):uint(), string.format("Subcmd reply: 0x02 [Device info], Right Joy-Con, FW Version: %X.%02x, MAC Addr: %02x:%02x:%02x:%02x:%02x:%02x", buf(24,1):uint(), buf(25,1):uint(), buf(28,1):uint(), buf(29,1):uint(), buf(30,1):uint(), buf(31,1):uint(), buf(32,1):uint(), buf(33,1):uint()))
+        end
+      elseif switch_subcmdack == 0x9010 then
+        pkt.cols.info = "SPI Read reply"
+        subtree:add(f_subcommandjoy, buf(23,1), buf(23,1):uint(), string.format("Subcmd reply: 0x10 [SPI Read reply]"))
+      elseif switch_subcmdack == 0x8008 then
+        pkt.cols.info = "Shipment set OK"
+        subtree:add(f_subcommandjoy, buf(23,1), buf(23,1):uint(), string.format("Subcmd reply: 0x08 [Shipment set OK]"))
+      elseif switch_subcmdack == 0x8003 then
+        pkt.cols.info = "Data Format set OK"
+        subtree:add(f_subcommandjoy, buf(23,1), buf(23,1):uint(), string.format("Subcmd reply: 0x03 [Data Format set OK]"))
+      elseif switch_subcmdack == 0x8040 then
+        pkt.cols.info = "6-Axis Enable set OK"
+        subtree:add(f_subcommandjoy, buf(23,1), buf(23,1):uint(), string.format("Subcmd reply: 0x40 [6-Axis Enable set OK]"))
+      elseif switch_subcmdack == 0x8048 then
+        pkt.cols.info = "Vibration Enable set OK"
+        subtree:add(f_subcommandjoy, buf(23,1), buf(23,1):uint(), string.format("Subcmd reply: 0x48 [Vibration Enable set OK]"))
+      elseif switch_subcmdack == 0x8030 then
+        pkt.cols.info = "Player Lights set OK"
+        subtree:add(f_subcommandjoy, buf(23,1), buf(23,1):uint(), string.format("Subcmd reply: 0x30 [Player Lights set OK]"))
+      elseif switch_subcmdack == 0xA021 then
+        pkt.cols.info = "MCU Config Write reply"
+        subtree:add(f_subcommandjoy, buf(23,1), buf(23,1):uint(), string.format("Subcmd reply: 0x21 [MCU Config Write reply], Arg: %08x %08x %08x %08x %08x %08x %08x %08x %08x %02x", buf(24,4):uint(), buf(28,4):uint(), buf(32,4):uint(), buf(36,4):uint(), buf(40,4):uint(), buf(44,4):uint(), buf(48,4):uint(), buf(52,4):uint(), buf(52,4):uint(), buf(57,1):uint()))
+      elseif switch_subcmdack == 0x8304 then

```

Comment:
Yes. Check my other changes also. There are a lot new things going on. For example the vibration algorithm and freq/amp tables.

I found more but did not have the time to update the repo. Now I'm on vacations so maybe I will do sth about it.